core/arch/arm/kernel/static_ta.c

```diff
@@ -130,7 +130,12 @@ static void static_ta_enter_close_session(struct tee_ta_session *s)
 
 static void static_ta_destroy(struct tee_ta_ctx *ctx __unused)
 {
-	/* Nothing to do */
+	if(ctx){
+		struct static_ta_ctx *stc = to_static_ta_ctx(ctx);
+		if(stc){
```

Comment:
Even if it's a NULL pointer it's perfectly fine to just call `free(stc)`, i.e. you don't need the `if (stc)` check.